LibUsbgxConfig.cmake.in

```diff
@@ -0,0 +1,18 @@
+get_filename_component(LibUsbgx_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+include(CMakeFindDependencyMacro)
+
+list(APPEND CMAKE_MODULE_PATH ${LibUsbgx_CMAKE_DIR})
+list(REMOVE_AT CMAKE_MODULE_PATH -1)
+
+set(LibUsbgx_INCLUDE_DIRS "@includedir@")
+
+if(NOT TARGET LibUsbgx::LibUsbgx)
+	add_library(LibUsbgx::LibUsbgx SHARED IMPORTED)
+	set_property(TARGET LibUsbgx::LibUsbgx APPEND PROPERTY IMPORTED_CONFIGURATIONS RELEASE)
+
+	set_target_properties(LibUsbgx::LibUsbgx PROPERTIES 
+		IMPORTED_LOCATION "@libdir@"
```

Comment:
Hmmm I took a look at libusbgx.pc.in and it seems that the easiest way to get this working would be to just declare cmake variables like in libusbgx.pc.in:

prefix=@prefix@
exec_prefix=@exec_prefix@
libdir=@libdir@
includedir=@includedir@

Reply:
Yeah, this somehow did the trick.